packages/storefront/README.md

```diff
@@ -62,3 +62,26 @@ var app = new EmberApp({
 });
 ```
 
+## Testing 
+
+```bash
+npm install
+bower install
+# current Ember version
+ember test
+# all supported Ember versions
+npm test
+```
+
+## Contributing 
+
+In the addon folder
+```bash
+npm link
+```
+
+In your project folder
+```bash
+npm link spree-ember-storefront
+ember generate spree-ember-storefront
+```
```

Comment:
Absolutely, actually I ran into some issues by using `npm link`... So, if you know a better way, please copy and edit. Just post a link to a PR or commit when its done, so I can delete this from here.